beta/experience-platform-extension/tutorials/tutorial-3-xdm-implementation.md

```diff
@@ -0,0 +1,286 @@
+# XDM implementation
+
+{% hint style="warning" %}
+The Adobe Experience Platform Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more and get access to the materials for this tutorial.
+{% endhint %}
+
+## Prerequisites for this tutorial
+
+* Access to Adobe Experience Platform
+* Access to AEP Real-time Customer Profile
+* Minimal Swift / Android development knowledge 
+* Knowledge about the AEP Edge extension
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup).
+
+### Download the sample application
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+Clone or download the Android Sample application from [https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3](https://github.com/adobe/aepsdk-sample-app-android/tree/beta-assignment-3).
```

Comment:
Add code sample on how to clone the specific tag/branch:
`git clone git@github.com:adobe/aepsdk-sample-app-android.git --branch beta-assignment-1 --single-branch`

For downloading the zip, If we are using tags, we should link to the tag and point out the download link.
https://github.com/adobe/aepsdk-sample-app-android/releases/tag/beta-assignment-1